patientsearch/src/js/helpers/utility.js

```diff
@@ -468,7 +468,8 @@ export function isInPast(dateString) {
   const today = new Date();
   const targetDate = new Date(dateString);
   if (!isValid(targetDate)) return false;
-  return targetDate < today;
+  const diff = (today - targetDate); // in miniseconds
+  return diff > (1000 * 60 * 5); // this will check if diff is 5 minutes or more
```

Comment:
i'd love a comment that explains the buffer, i.e.:

> pad by 5 mins to give system time to transmit message, rather than indicate no next message time during processing